src/styles.css

```diff
@@ -284,7 +349,7 @@ img.notion-page-icon {
 }
 
 .notion-asset-wrapper {
-  margin: 2px 0 0.5rem;
+  margin: 0.5rem auto 0.5rem;
```

Comment:
this is important to center images as in this example page https://demo.notion2site.com/d9ae0c6e7cad49a78e21d240cf2e3d04